package.json

```diff
@@ -18,21 +18,36 @@
     "url": "https://github.com/metaplex-foundation/js.git"
   },
   "scripts": {
-    "clean": "lerna run clean",
-    "build": "lerna run build",
-    "test": "lerna run test",
+    "clean": "nx run-many --all --target=clean",
+    "build": "nx run-many --all --target=build",
+    "test": "nx run-many --all --target=test",
+    "lint:test": "yarn workspaces run lint:test",
+    "lint:fix": "yarn workspaces run lint:fix",
+    "format": "prettier --write packages/",
     "docs": "typedoc",
     "deploy:docs": "yarn docs && gh-pages --dist docs --dotfiles",
-    "publish": "lerna publish from-package",
+    "bump:version": "yarn changeset add",
+    "commit:version": "yarn changeset version",
     "amman:start": "DEBUG='amman:(info|error|debug)' amman start",
-    "amman:stop": "amman stop",
-    "lint": "prettier -c .",
-    "lint:fix": "prettier --write ."
+    "amman:stop": "amman stop"
   },
   "devDependencies": {
     "@babel/core": "^7.17.10",
     "@babel/preset-env": "^7.17.10",
     "@babel/preset-typescript": "^7.16.7",
+    "@changesets/cli": "^2.24.4",
+    "@metaplex-foundation/amman": "^0.12.0",
+    "@metaplex-foundation/amman-client": "^0.2.2",
+    "@metaplex-foundation/beet": "^0.6.1",
+    "@metaplex-foundation/beet-solana": "^0.3.1",
+    "@nrwl/cli": "14.7.5",
+    "@nrwl/cypress": "14.7.5",
```

Comment:
Can you please only add the dependencies that we need? For instance, we're not using cypress so I'm not sure why we need to add this plugin on the dev dependencies. Can you please review each one and make sure it's being used otherwise remove it.

Reply:
checked-removed unused